src/main/java/com/rakovets/course/java/core/practice/io/FileAnalyzeUtil.java

```diff
@@ -0,0 +1,200 @@
+package com.rakovets.course.java.core.practice.io;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FileAnalyzeUtil {
+    public static List<String> getListOfStringFromFile(Path filePath) {
+        List<String> strings = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(String.valueOf(filePath)))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                strings.add(line);
+            }
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+        return strings;
+    }
+
+    public static List<String> getListOfWordsStartWithVowelLetterFromFile(Path filePath) {
+        List<String> tempList = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(String.valueOf(filePath)))) {
+            String line;
+            String[] strings;
+            while ((line = bufferedReader.readLine()) != null) {
+                strings = line.replaceAll("[\\[\\],.!?]", "").split(" ");
+                Arrays.stream(strings)
+                        .filter(e -> e.matches("^[eEuUiIoOaA].*"))
+                        .forEach(tempList::add);
+            }
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+        return tempList;
+    }
+
+    public static List<String> getListOfWordsEndsWithTheFirstLetterOfTheNextWord(Path filePath) {
+        List<String> listOfWords = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(String.valueOf(filePath)))) {
+            String line;
+            String[] strings;
+            while ((line = bufferedReader.readLine()) != null) {
+                strings = line.replaceAll("[,?!] ", "").split(" ");
+                for (int i = 0; i < strings.length - 1; i++) {
+                    char[] tempOne = strings[i].toCharArray();
+                    char[] tempTwo = strings[i + 1].toCharArray();
+                    if (tempOne[tempOne.length - 1] == tempTwo[0]) {
+                        listOfWords.add(strings[i]);
+                    }
+                }
+            }
+        } catch (IOException e){
+            System.out.println(e.getMessage());
+        }
+        return listOfWords;
+    }
+
+        public static List<String> getLargestAscendingNumbersOrder(Path filePath) {
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"